src/Spelling/WordParser.cs

```diff
@@ -0,0 +1,566 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="WordParser.cs" company="https://github.com/StyleCop">
+//   MS-PL
+// </copyright>
+// <summary>
+//   The word parser.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Spelling
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.ComponentModel;
+    using System.Text;
+
+    /// <summary>
+    /// The word parser.
+    /// </summary>
+    public class WordParser
```

Comment:
We already have a WordParser in our repo: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/WordParser.cs. How is the new one different from that?

Reply:
No longer applies